src/main/java/ch/uzh/ifi/hase/soprafs23/controller/UserController.java

```diff
@@ -135,4 +134,20 @@ public void updateUser(@PathVariable Long id,
 
       userService.updateUser(id, userPutDTO);
   }
+    @GetMapping("/users/{userId}")
```

Comment:
pls update REST table in teams according to the newly implemented calls. We introduced the "done" column for this.